problems/isOdd.js

```diff
@@ -7,8 +7,14 @@
 
 */
 
-function isOdd() {
-
+function isOdd(n) {
+return n.every((num) => {
```

Comment:
.every is an array method. n  is just a number. We want to see if the number is odd. 
return num % 2 === 1